foil.tex

```diff
@@ -10,16 +10,12 @@
 % chktex-file 37
 
 \documentclass{turgon}
-
-%\usepackage{lmodern}
-
 \usepackage[printwatermark]{xwatermark}
 \usepackage{tikz}
 \usetikzlibrary{shapes.geometric, arrows, positioning}
-\newwatermark[allpages,color=black!15,angle=55,scale=5,xpos=0,ypos=0]%
+\newwatermark[allpages, color = black!15, angle = 55, scale = 5, xpos = 0, ypos = 0]%
```

Comment:
Some tex compilers or packages did not allow arguments (in `[...]`).  We'll see.

Reply:
Sorry, I don't understant what it means. Do I have to revert it to `\newwatermark[allpages,color=black!15,angle=55,scale=5,xpos=0,ypos=0]%` ?

Reply:
That was my comment, not a request.